examples/v2/gke/istio/README.md

```diff
@@ -0,0 +1,13 @@
+# Sample DM template for GKE and istio.io
+
+Sample DM template that will initialize an **ALPHA** GKE cluster and add [istio.io](https://istio.io)
+
+
+>> NOTE: You must set your default compute service account to include:
```

Comment:
This looks pretty funny in the GitHub rendering of markdown. Are you sure you want it too look like that?